src/appshell/view/firstlaunchsetup/firstlaunchsetupmodel.cpp

```diff
@@ -0,0 +1,126 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#include "firstlaunchsetupmodel.h"
+
+using namespace mu::appshell;
+
+FirstLaunchSetupModel::FirstLaunchSetupModel(QObject* parent)
+    : QObject(parent)
+{
+    m_pages = {
+        Page { "ThemesPage.qml", "musescore://notation" },
+        Page { "PlaybackPage.qml", "musescore://notation", true },
```

Comment:
It would be great to add a comment on what this bool means (or even replace it with a static constant)

Reply:
Added a comment.
(See the first commit of #9803)